index.js

```diff
@@ -1,11 +1,31 @@
 var fs = require('fs-extra');
 var path = require('path');
 var run = require('electron-installer-run');
-var zipFolder = require('zip-folder');
-var format = require('util').format;
+var archiver = require('archiver');
 var series = require('async').series;
 var debug = require('debug')('electron-installer-zip');
 
+function zipFolder(srcFolder, zipFilePath, callback) {
+	var output = fs.createWriteStream(zipFilePath);
+	var zipArchive = archiver('zip');
+
+	output.on('close', function() {
+		callback();
+	});
+
+	zipArchive.pipe(output);
+
+	zipArchive.bulk([
```

Comment:
Note that `archiver('zip').bulk()` [is deprecated](https://github.com/archiverjs/node-archiver/pull/202), so this will probably need to be updated sooner or later.